### PR TITLE
Documentation on standalone installer and version tag for csi unity driver

### PR DESCRIPTION
**Pull request by karthikk92**
# Description
Documentation on standalone installer and vtag for csi unity driver

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/243 |

# Checklist:

- [] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/installation/helm/unity.md [hunk 1]
```diff
@@ -295,8 +295,14 @@ Procedure
     At the end of the script unity-controller Deployment and DaemonSet unity-node will be ready, execute command `kubectl get pods -n unity` to get the status of the pods and you will see the following:
     
     * One or more Unity Controller (based on controllerCount) with 5/5 containers ready, and status displayed as Running.
-    * Agent pods with 2/2 containers and the status displayed as Running. 
-    
+    * Agent pods with 2/2 containers and the status displayed as Running.
+
+**Note**:
+   To install nightly or latest csi driver build using bash script use the bellow command:
```
Comment (shanmydell) [inline content/docs/csidriver/installation/helm/unity.md:301]: below* typo

**Review by shanmydell — CHANGES_REQUESTED** [review]

**Comment by shanmydell** [comment]
@rajendraindukuri @karthikk92 : Pls get PR to closure

File: content/docs/csidriver/installation/helm/unity.md (hunk 1 above)
Comment (karthikk92) [reply content/docs/csidriver/installation/helm/unity.md]: Taken care

**Comment by karthikk92** [comment]
> @rajendraindukuri @karthikk92 : Pls get PR to closure

Taken care

**Review by shanmydell — APPROVED** [review]

**Review by panigs7 — APPROVED** [review]

File: content/docs/csidriver/installation/helm/unity.md [hunk 2]
```diff
@@ -295,8 +295,14 @@ Procedure
...
-    * Agent pods with 2/2 containers and the status displayed as Running. 
-    
+    * Agent pods with 2/2 containers and the status displayed as Running.
+
+**Note**:
+   To install nightly or latest csi driver build using bash script use the below command:
+      `/csi-install.sh --namespace unity --values ./myvalues.yaml --version nightly/latest`
+
+8. You can also install the driver using standalone helm chart using the command `helm install --values  myvalues.yaml --namespace unity unity ./csi-unity`
+
```
Comment (rensyct) [inline content/docs/csidriver/installation/helm/unity.md:305]: @karthikk92 , I feel Point 8 can be more descriptive and can be same as Point No 4  in PR https://github.com/dell/csi-unity/pull/82/files

**Pull request merged**